roles/cobbler/defaults/main.yml

```diff
@@ -27,3 +28,8 @@ ssh_keys: []
 
 ansible_user: ''
 ansible_user_uid_: ''
+
+# A list of lines to add to resolv.conf and resolv.conf.d/base
+resolvconf:
+  - "nameserver"
+  - "search"
```

Comment:
I'll default this to an empty list and write docs with an example of how to use it.